src/Compilers/Test/Utilities/VisualBasic/Microsoft.CodeAnalysis.VisualBasic.Test.Utilities.vbproj

```diff
@@ -38,6 +38,7 @@
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.VisualBasic.Scripting.UnitTests" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.VisualBasic.EditorFeatures.UnitTests" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.Rebuild.UnitTests" />
+    <InternalsVisibleTo Include="Microsoft.CodeAnalysis.CSharp.Symbol.UnitTests" />
```

Comment:
><InternalsVisibleTo Include="Microsoft.CodeAnalysis.CSharp.Symbol.UnitTests" /> [](http://example.com/codeflow?start=4&length=79)

Consider removing and using an alternative approach to `FindBindingText` from the C# test.